### PR TITLE
chore: excise psql-extra, django-localized-fields

### PR DESCRIPTION
**Pull request by japsu**

### CODE REVIEW

File: backend/program_v2/models/offer_form.py [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
     event = models.ForeignKey("core.Event", on_delete=models.CASCADE, related_name="program_offer_forms")
     slug = models.CharField(**NONUNIQUE_SLUG_FIELD_PARAMS)  # type: ignore
 
-    short_description = LocalizedTextField(
-        blank=True,
-        default=dict,
-        verbose_name=_("short description"),
-        help_text=_("Visible on the page that offers different kinds of forms."),
-    )
+    short_description = HStoreField(blank=True, default=dict)
```
Comment (akx) [inline backend/program_v2/models/offer_form.py:14]: Was the help text yote on purpose?

File: backend/tickets/models/tickets_event_meta.py [hunk 2]
```diff
@@ -134,13 +133,7 @@ class TicketsEventMeta(ContactEmailMixin, EventMetaBase, LocalizedModel):
...
     )
 
-    terms_and_conditions_url = LocalizedCharField(
-        blank=True,
-        default=dict,
-        max_length=1023,
-        verbose_name=_("Terms and conditions URL"),
-        help_text=_("If set, customers will be required to indicate acceptance to finish order."),
-    )
+    terms_and_conditions_url = HStoreField(blank=True, default=dict)
```
Comment (akx) [inline backend/tickets/models/tickets_event_meta.py:136]: Ditto for help text here. Also, wonder what the semantics for the default are - is it a default for each key..?

**Review by akx — COMMENTED** [review]
Some hmms from the couch.

File: backend/tickets/models/tickets_event_meta.py (hunk 2 above)
Comment (japsu) [reply backend/tickets/models/tickets_event_meta.py:136]: No, it's a default for the whole field.

File: backend/program_v2/models/offer_form.py (hunk 1 above)
Comment (japsu) [reply backend/program_v2/models/offer_form.py:14]: Actually no! I simply replaced every LocalizedField with the same HStoreField copypasta to quickly try it out. I'll add the mistakenly yought help texts back.

**Pull request merged**